test/transform/output.js

```diff
@@ -0,0 +1,42 @@
+var _templateReference = {},
+  _templateReference2 = {},
+  _templateReference3 = {},
+  _templateReference4 = {};
+const div = ESXToken.template(
+  _templateReference,
+  ESXToken.element("div", null)
+);
+const div2 = ESXToken.template(
+  _templateReference2,
+  ESXToken.element(
+    "div",
+    ESXToken.create(ESXToken.MIXED_TYPE, [
+      ESXToken.property(ESXToken.STATIC_TYPE, "a", "a"),
+      ESXToken.property(ESXToken.RUNTIME_TYPE, "b", "b"),
+    ]),
+    ESXToken.create(
+      ESXToken.STATIC_TYPE,
+      ESXToken.element("p", null, ESXToken.create(ESXToken.STATIC_TYPE, "c"))
+    )
+  )
+);
+function MyComponent(...args) {
+  return ESXToken.template(
+    _templateReference3,
+    ESXToken.factory(
```

Comment:
not sure about where this is defined in the `ESXToken` class 🤔

Reply:
Nowhere, it should be `fragment`